### PR TITLE
Grants: fix Plain

### PR DESCRIPTION
**Pull request by estyxx**
- Use email as Customer identifier
- Add Sentry to aws lambda

### CODE REVIEW

File: backend/wsgi_handler.py [hunk 1]
```diff
@@ -24,6 +27,21 @@
 logger = getLogger(__name__)
 
 
+SENTRY_DSN = settings.SENTRY_DSN
+
+if SENTRY_DSN:
```
Comment (marcoacierno) [inline backend/wsgi_handler.py:32]: is this present somewhere else too?

Comment (estyxx) [reply backend/wsgi_handler.py:32]: settings.prod

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**